packages/terra-signature/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "terra-signature",
+  "main": "lib/Signature.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "{insert description}",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-signature",
+    "Signature",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.x"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^0.x",
+    "terra-mixins": "^1.0.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "jquery": "^3.2.1",
```

Comment:
Introducing jQuery will probably require some discussion. Can you give a short explanation on where and why this is needed? 

The versioning and introduction of this could potentially cause thrashing with legacy applications in the process of uplifting.

Reply:
The initial version of the code used jQuery but I replaced it based on feedback from Dave Kasper.  Will remove the dependency, it is no longer needed.